src/ios/ListLibraryItems.m

```diff
@@ -167,7 +175,7 @@ - (void)uploadItem:(CDVInvokedUrlCommand *)command {
 
                             [request setValue:[[[NSFileManager defaultManager] sizeOfItemAtURL:mLocalTempURL] stringValue] forHTTPHeaderField:@"Content-Length"];
                             [request setValue:[[NSFileManager defaultManager] md5OfItemAtURL:mLocalTempURL] forHTTPHeaderField:@"Content-MD5"];
-                            NSURLSessionUploadTask * task = [session uploadTaskWithRequest:request fromFile:mLocalTempURL];
+                            NSURLSessionTask *task = [session uploadTaskWithRequest:request fromFile:mLocalTempURL];
```

Comment:
Why this change?

Reply:
It was a typo, my bad. I just fixed it in the last commit.